redbot/core/drivers/sql/sql.py

```diff
@@ -0,0 +1,246 @@
+import asyncio
+import concurrent
+import ujson as json
```

Comment:
so this needs to be looked at im really tempted to make ujson a core dependency, but if not then this can just be changed to normal json.